cea/optimization/supportFn.py

```diff
@@ -64,9 +64,8 @@ def calcQmax(fName, filepath, gV):
     :return: Qmax: maximum heating power [W]
     :rtype: float
     """    
-    HeatfeatureList = ["T_sst_heat_return_netw_total", "T_sst_heat_supply_netw_total", "mdot_DH_netw_total"]
-    buildDemand = extractDemand(filepath+'//'+fName, HeatfeatureList, gV.DAYS_IN_YEAR)
-    Qmax = np.max(buildDemand)
+    Q_DHNf_W = pd.read_csv(filepath + '//' + fName, usecols=["Q_DHNf_W"]).values
```

Comment:
@daren-thomas: It did work. I just verified again. But I agree that `fName` and `file_name` are not PEP8. Stupid Bhargav. Will modify :)